docs_src/async_sql_databases/sql_app/main.py

```diff
@@ -0,0 +1,67 @@
+from typing import List
+
+from fastapi import Depends, FastAPI, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from . import crud, schemas
+from .database import Base, engine
+
+app = FastAPI()
+
+
+@app.on_event("startup")
+async def start_db():
+    async with engine.begin() as conn:
+        # await conn.run_sync(Base.metadata.drop_all)
+        await conn.run_sync(Base.metadata.create_all)
+
+
+# Dependency
+async def get_db():
+    session = AsyncSession(engine)
```

Comment:
Needed due to Pydantic.
```suggestion
    session = AsyncSession(engine, expire_on_commit=True)
```

Reply:
Not really sure what that does, but I'll just trust you.